.bowerrc

```diff
@@ -0,0 +1,6 @@
+{
+  "ignoredDependencies": [
+    "bootstrap",
+    "bootstrap-sass-official"
```

Comment:
hm.. I see. I didn't know that. It's better to remove `bootstrap-sass-official`.

BTW, I missed to install `bootswatch` with `--save-dev`. 
Installing bootswatch with `--save-dev` didn't insert `bootstrap.js` twice during `wiredep`. So I didn't have to ignore anything.

I will remove `.bowerrc` and install `bootswatch` in `devDependencies` again.
I'm sorry for the inconvenience.